### DISCUSSION TITLE
eOS 8. Open pdf's with Document Viewer

### DISCUSSION THREAD

**Discussion by garzajual** [elementary/triage#637] [Q&A]
I just installed eOS 8. Why don't pdf files open with Document Viewer instead of ImageMagick. It is easy to change that behavior, but the logical thing to do would be to use the Document Viewer.

**Comment by ryonakano** [comment]
We don't ship ImageMagick by default on elementary OS 8, so PDF files will be opened with Document Viewer.

I guess you installed ImageMagick manually or alongside with another apps that depends on it and possibly that made ImageMagick the default app to open PDF files.
